lib/constructor.js

```diff
@@ -234,6 +234,11 @@ const Sharp = function (input, options) {
     pngQuality: 100,
     pngBitdepth: 8,
     pngDither: 1,
+    jp2Quality: 80,
+    jp2TileHeight: 256,
```

Comment:
libvips has default tile dimensions of 512 - I'd prefer to keep things consistent unless there's a reason for the lower value.

Reply:
There is no particular reason it was while I was performing my tests I forgot to reset it to 512 😅